pom.xml

```diff
@@ -512,11 +516,12 @@
                 <artifactId>license-maven-plugin</artifactId>
                 <version>${license-maven-plugin.version}</version>
                 <configuration>
-                    <header>com/mycila/maven/plugin/license/templates/MIT.txt</header>
+                    <header>${maven.multiModuleProjectDirectory}/LICENSE.md</header>
+                    <aggregate>true</aggregate>
                     <properties>
                         <owner>Ilkka Seppälä</owner>
                     </properties>
-                    <skipExistingHeaders>true</skipExistingHeaders>
+                    <skipExistingHeaders>false</skipExistingHeaders>
```

Comment:
Why this has been set as false?

Reply:
@ohbus 
This will update all the headers in our project with the latest license file.
But based on your comments I guess I will revert this change.

Reply:
Sounds good.